marklogic-data-hub-central/ui/e2e/cypress/integration/explore/scenarios.queries.tsx

```diff
@@ -677,7 +677,9 @@ describe("manage queries modal scenarios on detail page", () => {
     cy.waitForAsyncRequest();
     browsePage.waitForSpinnerToDisappear();
     cy.waitUntil(() => browsePage.getManageQueriesButton().should("have.length.gt", 0));
-    browsePage.getManageQueriesButton().should("be.visible");
+    cy.waitForAsyncRequest();
+    cy.wait(1000);
+    cy.waitUntil(() => browsePage.getManageQueriesButton().should("be.visible"));
```

Comment:
@timur-isangulov We are already waiting until we see getManageQueriesButton on line 679. and instead of cy.wait(1000); Can we use something like cy.waitUntil(() => browsePage. getManageQueriesButton(), {timeout: 10000})?

Reply:
Yeah, let's test it on pipeline.